src/map/options.ts

```diff
@@ -106,6 +108,7 @@ export class MapOptions extends OptionsGroup {
                 throw Error(`size factor must be between 0 and 100, got ${value}`);
             }
         };
+        this.size.mode.validate = optionValidator(['', 'linear', 'log', 'sqrt', 'inverse'], 'size');
```

Comment:
why default to `''` instead of `linear`?

Reply:
I just added a push which designates "constant" as the marker size default, which is different than "linear"

Reply:
We already use `this.size.property` set to an empty string for constant size, this feels a bit redundant. Do you have a use case for having both options?

Reply:
wouldn't it make more sense to disable this.size.property in case of constant scaling versus the other way around?

Reply:
yep, that makes more sense!

Reply:
Let me know what you think of the changes